Python/syntax_test_python.py

```diff
@@ -1350,6 +1350,74 @@ class Starship:
 #        ^ punctuation.separator.annotation.variable.python
 #                                   ^ keyword.operator.assignment
 
+# Type comments - type: ignore must be by itself.
+
+primes = 5  # type: ignore # type: not-a-type-comment
+#           ^ comment.line.type-hint punctuation.definition.comment.python comment.line.number-sign.python
```

Comment:
The stacking of scopes here looks wrong. We shouldn't be adding a second `comment.line`, and especially not on top of the `punctuation.definition.comment`.